debugger/src/main/java/org/codice/acdebugger/api/ReflectionUtil.java

```diff
@@ -282,10 +282,7 @@ public Method findMethod(ReferenceType type, String name, String signature) {
     if (obj == null) {
       return null;
     }
-    final ReferenceType type =
-        (obj instanceof ClassObjectReference)
-            ? ((ClassObjectReference) obj).reflectedType()
-            : obj.referenceType();
+    final ReferenceType type = obj.referenceType();
```

Comment:
ℹ️ The original code was to handle static fields which are now handled separately with `getStatic()`. It also created issues when actually trying to access static field of the class `Class`.